app/models/guest_group.rb

```diff
@@ -5,6 +5,14 @@ def id_and_subgroup_ids
     Array(id)
   end
 
+  def subgroups
+    Group.none
+  end
+
+  def documents
+    Document.none
```

Comment:
Hmmmm I wonder if we want to move the `has_many :documents` line from FormalGroup to Group instead of doing this.

Reply:
done